logger.js

```diff
@@ -24,7 +24,7 @@ function pinoLogger (opts, stream) {
   loggingMiddleware.logger = logger
   return loggingMiddleware
 
-  function onResFinished (err) {
+  function onResFinished (err, msg = 'completed') {
```

Comment:
`pino-http` supports node v4+, and this syntax is available only on node v6+. Moreover, it is slower than using `||` to set a default.